isc24_tutorial/isc24_tutorial_proposal.md

```diff
@@ -0,0 +1,164 @@
+|11:00-11:30 | Break|
+|11:30-12:10 | Lesson design|
+|12:10-13:00 | Good interactive teaching practices|
+|13:00-14:00 | Lunch break|
+|14:00-14:40 | Teaching mechanics|
+|14:40-15:20 | Collaborative lesson development|
+|15:20-15:30 | Break|
+|15:30-16:00 | Practice: teaching and giving feedback|
+|16:00-16:30 | Break|
+|16:30-17:30 | Practice: teaching and giving feedback|
+|17:30-18:00 | Recap, final discussion and where to go from here|
+
+**Description of hands-on activities, if applicable**
+
+The tutorial will include ample time for a variety of hands-on activities.
+Participants be split into groups already during the first morning session
+where they take turns to teach a short snippet of a technical topic to each
+other - e.g. the Unix command line - followed by providing feedback to each
+other. This experience will form a baseline. In following sessions,
+participants will be split into groups of varying sizes to collectively
+brainstorm and discuss pedagogical concepts or provide feedback to recorded
+teaching demonstrations.  There will also be technical exercises where
+participants learn how to use the Sphinx Python package to develop training
+lessons in a team using collaborative workflows in Git.  In the final hands-on
+activity, participants will again teach a topic to each other in groups and
+provide feedback, and reflect on how the concepts learned during the tutorial
+have affected their teaching compared to the baseline.
+
+**Updates from previous presentations, if applicable**
+
+This tutorial was taught over two half days to around 30 participants in an
+online training workshop targeting the EuroCC network on September 28-29, 2023
+(https://enccs.se/events/best-practices-in-online-and-in-person-training/). The
+feedback from this workshop was exceptionally positive, with an average rating
+of over 9 out of 10 and 100% of participants reporting that they would
+recommend the training to their colleagues.  For the ISC tutorial, some of the
+exercises will be adapted to the in-person setting, and other minor
+improvements will be made to further improve the flow of the training.
+
+**Number of attendees from previous tutorials, if applicable**
+
+Around 30 participants attended the online workshop delivered on September
+28-29 2023. This will be the first time the tutorial is taught in-person at
+ISC.
+
+**URLs to sample slides and other material**
+
+The training material - which will be somewhat updated before the tutorial -
+can be found at https://enccs.github.io/instructor-training/.
+
+## Logistics
+
+**Length of the tutorial: half-day (3.5 hours) or full-day (7 hours)**
+
+full-day 7 hours
+
+**Percentage of content split as beginner/intermediate or intermediate/advanced**
+
+50% beginner/intermediate, 50% intermediate/advanced
+
+
+**Requirements for attendees (e.g., laptop, remote server account)**
+
+- laptop
+- (optional, maybe) Python installation with some libraries, alternatively cloud
```

Comment:
Why the Python? Is this for Sphinx?

Reply:
yes this is for doing the sphinx exercises in https://enccs.github.io/instructor-training/collaborative-development/

but everything can be done in the browser and of course it'll be simpler for us and for the participants to not have to worry about local python, sphinx and git installations. Is that the best way to go? Shall we remove the requirements?